resize-observer/devicepixel2.html

```diff
@@ -0,0 +1,163 @@
+<!doctype html>
+<link rel="match" href="devicepixel2-ref.html">
+<meta name="assert" content="Device pixel content box sizes and pixel snapping are correct in Resize Observer callback">
+<style>
+  html, body {
+    margin: 0;
+  }
+  #front {
+    position: absolute;
+    left: 0;
+    top: 0;
+  }
+  .outer {
+    width: 100px;
+    display: flex;
+  }
+  .outer>* {
+    flex: 1;
+    height: 20px;
+  }
+</style>
+<body>
+  <div id="front"></div>
+</body>
+<script>
+const patternSize = 4;
+
+function createPatternDataURL() {
+  const ctx = document.createElement('canvas').getContext('2d');
+  ctx.canvas.width = patternSize;
+  ctx.canvas.height = patternSize;
+
+  const b = [0, 0, 0, 255];
+  const t = [0, 0, 0, 0];
+  const r = [255, 0, 0, 255];
+  const g = [0, 255, 0, 255];
+
+  const imageData = new ImageData(4, 4);
```

Comment:
s/4/patternSize/ here maybe, since we've got that named constant?

Reply:
done